style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blueviolet;
+}
```

Comment:
- [ ] Please make sure that you are using the class with styles for your h1 instead of targeting it directly over the tag name.